metrics/src/test/java/com/newrelic/telemetry/json/TypeDispatchingJsonTelemetryBlockWriterTest.java

```diff
@@ -1,19 +1,45 @@
 package com.newrelic.telemetry.json;
 
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
+import com.newrelic.telemetry.Metric;
+import com.newrelic.telemetry.Span;
+import com.newrelic.telemetry.Telemetry.Type;
+import com.newrelic.telemetry.TelemetryBatch;
 import org.junit.jupiter.api.Test;
 
 class TypeDispatchingJsonTelemetryBlockWriterTest {
```

Comment:
Add test for the unsupported `batch.getType()`

Reply:
Due to the nature of the enum, this is very challenging and I don't think it's worth doing.  If you have an interesting way of getting coverage, I'd love to hear it!